shelley/chain-and-ledger/formal-spec/multi-sig.tex

```diff
@@ -511,86 +566,94 @@ \subsection{Delegation Specific Changes}
 \end{figure}
 
 \textbf{Note:} In contrast to staking reference delegation, staking pools
-themselves cannot use multi-signature schemes. Else, the lightweight
-certificates for delegation from the pool to the KES hot key would have to be
-script witnesses. These certificates have to be included in each header, but
-those headers are required to have a fixed size and be as small as possible.
+themselves cannot use multi-signature schemes. Otherwise, the lightweight
+certificates that are used for delegation from the pool to the KES hot keys would also need to be
+script witnesses. This is undesirable since these certificates need to be included in each header,
+but headers are required to have a minimal and fixed size.\khcomment{What size is acceptable?}
 
 \subsection{UTXOW Transition Rule}
 \label{sec:utxow-transition}
 
-The UTXOW extended transition system is shown in
-Figure~\ref{fig:rules:utxow-multi-sig}. The constraint on the set of required
+The UTXOW extended transition system\khcomment{Refer to location in original document} is shown in
+Figure~\ref{fig:rules:utxow-multi-sig}
+\khcomment{Something has gone wrong with ``inference''.  Also, the layout is poor - don't split conditions over 2 lines, or lay them out as centred with each other.}. The constraint on the set of required
 witnesses is relaxed in such a way that \emph{redundant} signatures can be
-supplied in the transaction. The set of verification keys is passed to the
+supplied in the transaction.\khcomment{redundant meaning ``not required'' or ``duplicate''?} The set of verification keys is passed to the
 validator script via the concrete implementation of $\fun{validateScript}$ for
 the specific script type.
-
+%
 The set of all validator scripts of $\fun{txinsScript}~(\fun{txins}~tx)$ is
 checked for:
 
-\begin{itemize}
-\item the hashed script being equal to the hash stored in the output to spent,
-  and
-\item the validator script validating the transaction, and
-\item being equal to the set of the scripts required for the transaction (as
+\begin{enumerate}
+\item equality of the hashed script with the hash that is stored in the output to spent;
+\item that the script validates the transaction; and
+\item  that it is precisely\khcomment{or includes? - made a change in the figure} the set of the scripts required for the transaction (as
```

Comment:
here we do not allow more script witnesses than necessary.

Reply:
Is that part of the design?